cpp/ql/src/Security/CWE/CWE-611/XXE.qhelp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Parsing untrusted XML files with a weakly configured XML parser may lead to an
+XML External Entity (XXE) attack. This type of attack uses external entity references
```

Comment:
I'd suggest losing the caps, especially as they don't correspond to the initialism.

```suggestion
XML external entity (XXE) attack. This type of attack uses external entity references
```